pyproject.toml

```diff
@@ -25,5 +25,7 @@ manylinux-i686-image = "manylinux2014"
 
 [[tool.cibuildwheel.overrides]]
 select = "cp3?-*"
+# update skip when numpy wheels become available
+skip = ["*-musllinux_*", "cp310-win32"]
```

Comment:
This is an overrides section, just like mypy overrides, so it only applies on things that match the select statement, `cp3?-` (which is Python 3.6, 3.7, 3.8, and 3.9) - the skip should be placed a few lines above.

(The purpose of the override in this case is to use manylinux2010 on older Pythons, but leave 2014 for 3.10 and newer)

Reply:
Ok.

Reply:
If you find `select = "cp3{6,7,8,9}-*"` clearer, feel free to use that instead.

Reply:
Or add comments, I didn't notice and copy over the commend in boost-histogram: https://github.com/scikit-hep/boost-histogram/blob/0ddcc8db4007978f44051a8a32ef44fa23528689/pyproject.toml#L94